cip.go

```diff
@@ -345,6 +345,12 @@ func main() {
 		os.Exit(0)
 	}
 
+	// Check the pull request
+	pullRequestOK := sc.RunChecks(promotionEdges, []reg.PullCheck{})
```

Comment:
You should pass in a non-empty list of checks to iterate over.

Reply:
I'm passing in an anonymous list right now that can be filled with checks later on as they are added

Reply:
SG